src/game/variables.h

```diff
@@ -16,6 +16,7 @@ MACRO_CONFIG_INT(ClAutoswitchWeapons, cl_autoswitch_weapons, 0, 0, 1, CFGFLAG_CL
 MACRO_CONFIG_INT(ClShowhud, cl_showhud, 1, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Show ingame HUD")
 MACRO_CONFIG_INT(ClFilterchat, cl_filterchat, 0, 0, 2, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Show chat messages from: 0=all, 1=friends only, 2=no one")
 MACRO_CONFIG_INT(ClShowsocial, cl_showsocial, 1, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Show social data like names, clans, chat etc.")
+MACRO_CONFIG_INT(ClChatSound, cl_chatsound, 6, 0, 6, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Enable chat sound for 0=no msgs 1=whisper only 2=tagged 3=(1) & (2) 4=(3) & server msg 5=(4) & clientside msgs 6=all chat msgs")
```

Comment:
For the lambda user, I think this is very confusing. Maybe it would be better to split those? I don't know.

Reply:
128 visible characters, had not much spare space there, I guess.